src/utils/find-files.ts

```diff
@@ -0,0 +1,31 @@
+import R from 'ramda'
+import path from 'path'
+import glob from 'glob'
+
+import { getFileExtensionPattern } from './read-file'
+
+export const isNotSchema = (filePath: string): boolean =>
+  R.compose<string, string[], string, boolean>(
+    fileName => !fileName.startsWith('schema'),
```

Comment:
think positive :)

`isSchema` makes the code a bit easier IMO, because we already have a way of saying `not` with code in form of `!`:

```suggestion
    fileName => fileName.startsWith('schema'),
```

Reply:
Also: Do we want to leave the option to give different names to the schema? Currently `schemablablabla.json` would pass here.

```suggestion
    fileName => fileName === 'schema.json',
```

would be the safest option

Reply:
Regarding your fist comment: Seems a nit to me since we only check for the negated form. Now, the `!` is in `!isSchema`. However, I think i'm gonna replace `R.filter` with `R.reject` :)

Regarding your second comment: Yeah but i'd say we leave it for now as this will change once we can pass a configuration object. One of these option will be the schema path. So this will change anyway.

Reply:
regarding first comment: updated